c3dlas.c

```diff
@@ -911,20 +1010,20 @@ void boxSize2i(const AABB2i* b, Vector2i* out) {
 
 // BUG: needs some fancy math work to keep everything tight. integers don't split nicely
 void boxQuadrant2i(const AABB2i* in, char ix, char iy, AABB2i* out) {
-	Vector2 sz, c;
+	Vector2i sz, c;
 	
 	printf("fix me: %s:%d", __FILE__, __LINE__);
 	exit(666);
 	
-	boxCenter2(in, &c);
-	boxSize2(in, &sz);
+	boxCenter2i(in, &c);
+	boxSize2i(in, &sz);
 	sz.x *= .5;
```

Comment:
Backed out the Vector2 -> Vector2i changes. Ended up making boxSize2i output a Vector2 like the others (even though it should always return an integer size; it's only called from boxQuadrant2i currently).